udata_front/theme/gouvfr/datagouv-components/src/components/ResourceAccordion/EditButton.vue

```diff
@@ -9,20 +9,35 @@
 </template>
 
 <script setup lang="ts">
+import { computed } from "vue";
 import { config } from "../../config";
 import { useI18n } from "vue-i18n";
+import { CommunityResource, Resource } from "../../types/resources";
 
 type Props = {
   datasetId: string,
   isCommunityResource?: boolean,
-  resourceId: string,
+  resource: Resource | CommunityResource,
 };
 
 const props = withDefaults(defineProps<Props>(), {
   isCommunityResource: false,
 });
 
 const { t } = useI18n();
-const resourceType = props.isCommunityResource ? 'community-resource' : 'resource';
-const adminUrl = `${config.admin_root}dataset/${props.datasetId}/${resourceType}/${props.resourceId}`;
+
+const adminUrl = computed(() => {
+  if (config.new_admin) {
```

Comment:
Shouldn't it be `new_admin_url`?

Reply:
No it's a boolean indicating if we are using the new type of URLs (with the plural)

Reply:
If `VITE_NEW_ADMIN` is not set, we use the old URLs (for other users of udata/udata-front), but if we set it to `=true` we will use the plurals and the new links

Reply:
I understand your point, I added a default value to false in 64ed7088

Reply:
and then to `true` in https://github.com/datagouv/udata-front/pull/682/commits/b578a4e322a850212cfefbb17a8388c58dd12dc4 :ok_hand: